File: src/Omnia.ProcessManagement.Models/Processes/ProcessItem.cs
using Omnia.Fx.Models.Language;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Processes
{
    public class ProcessItem : Omnia.Fx.Models.JsonTypes.OmniaJsonBase
    {
        public enum ProcessItemTypes
        {
            Undefined = 0,
            Internal = 1,
            External = 2,
            CustomLink = 3
        }

        public virtual ProcessItemTypes Type { get; set; } = ProcessItemTypes.Undefined;

        public MultilingualProcessContentRef MultilingualProcessContentRef { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processes/ProcessItem.ts
import { GuidValue, LanguageTag, MultilingualString } from '@omnia/fx-models';

export enum ProcessItemTypes {
    Undefined = 0,
    Internal = 1,
    External = 2,
    CustomLink = 3
}

export interface ProcessItem {
    type: ProcessItemTypes;
    title: MultilingualString;
}